app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -50,4 +66,27 @@ class AdministratorControlsActivity :
       return KEY_NAVIGATION_PROFILE_ID
     }
   }
+
+  override fun loadProfileList() {
+    lastLoadedFragment = PROFILE_LIST_FRAGMENT
+    administratorControlsActivityPresenter
+      .setExtraControlsTitle(getString(R.string.administrator_controls_edit_profiles))
+    administratorControlsActivityPresenter.loadProfileList()
+  }
+
+  override fun loadAppVersion() {
+    lastLoadedFragment = APP_VERSION_FRAGMENT
+    administratorControlsActivityPresenter
+      .setExtraControlsTitle(getString(R.string.administrator_controls_app_version))
+    administratorControlsActivityPresenter.loadAppVersion()
+  }
+
+  override fun onSaveInstanceState(outState: Bundle) {
+    super.onSaveInstanceState(outState)
+    val titleTextView = findViewById<TextView>(R.id.extra_controls_title)
```

Comment:
This logic should be shifted to presenter. Try to keep `Activity` keep as much as possible.

Reply:
Done